juntagrico/entity/member.py

```diff
@@ -147,14 +147,17 @@ def subscriptions_old(self):
         return [sm.subscription for sm in
                 self.subscriptionmembership_set.filter(q_left_subscription())]
 
-    def join_subscription(self, subscription):
+    def join_subscription(self, subscription, primary=False):
         sub_membership = self.subscriptionmembership_set.filter(subscription=subscription).first()
         if sub_membership and sub_membership.leave_date:
             sub_membership.leave_date = None
             sub_membership.save()
         else:
             join_date = None if subscription.waiting else timezone.now().date()
             SubscriptionMembership.objects.create(member=self, subscription=subscription, join_date=join_date)
+        if primary:
```

Comment:
not sure if part of solution or has beeen accidentally here

Reply:
That came up while rewriting some tests. Collateral benefit :-)